test/unit/views/behaviours/ViewBehaviours.scala

```diff
@@ -49,13 +48,12 @@ trait ViewBehaviours extends ViewSpecBase {
           val doc = asDocument(view())
           for (key <- expectedGuidanceKeys) assertContainsText(doc, messages(s"$messageKeyPrefix.$key"))
         }
-
         "contain a timeout dialog" in {
-          val doc = asDocument(view())
+          val timeoutElm = asDocument(view()).select("meta[name=\"hmrc-timeout-dialog\"]")
           if(expectTimeoutDialog) {
-            assertRenderedById(doc, "timeoutDialog")
+            timeoutElm.size == 1
```

Comment:
is this check asserted?

Reply:
does `==` behave in the same way as `shouldBe` or another assertion? it looks like it'll just return `true` or `false` as opposed to failing the test

Reply:
nope, cleary not fully awake yet, will fix it